app/src/main/java/org/simple/clinic/summary/bloodpressures/newbpsummary/NewBloodPressureSummaryViewModel.kt

```diff
@@ -5,12 +5,16 @@ import java.util.UUID
 
 data class NewBloodPressureSummaryViewModel(
     val patientUuid: UUID,
-    val bloodPressures: List<BloodPressureMeasurement>?
+    val bloodPressures: List<BloodPressureMeasurement>?,
+    val bloodPressuresCount: Int
```

Comment:
This should be a nullable property, since it is not clear from a value of `0` if we have loaded the BPs and found there to be none, or if we have never loaded them at all.